### PR TITLE
TINY-8828: Prevent selection of cef content with the mouse on Firefox

### PR DESCRIPTION
**Pull request by vpyshnenko**
Related Ticket: TINY-8828

Description of Changes:
* add css rule to forbid the user to select the text inside of cef element
*  add css rule to retain the ability to select the content of cet element wrapped with cef element on Safari

I'm struggling to simulate dragging the mouse from an adjacent element to cef content, so probably this issue can be a candidate for manual testing by QA

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set

GitHub issues (if applicable):

### CODE REVIEW

**Comment by lnewson** [comment]
I've converted this to a draft as it doesn't look ready to review yet.

File: modules/oxide/src/less/theme/content/contenteditable/contenteditable.less [hunk 1]
```diff
@@ -33,9 +33,14 @@
...
   *[contentEditable=false] {
     cursor: default;
+    user-select: none;
   }
 
   *[contentEditable=true] {
     cursor: text;
   }
+
+  *[contentEditable=false] *[contentEditable=true] {
```
Comment (TheSpyder) [inline modules/oxide/src/less/theme/content/contenteditable/contenteditable.less:43]: Couldn't this just go in the `*[contentEditable=true]` block? `user-select` is not inherited so it should still work the same way `cursor` does.

- [ ] consider moving `user-select` into the `true` block

**Review by TheSpyder — APPROVED** [review]

File: modules/oxide/src/less/theme/content/contenteditable/contenteditable.less (hunk 1 above)
Comment (vpyshnenko) [reply modules/oxide/src/less/theme/content/contenteditable/contenteditable.less:43]: It seems there is an exception for Safari: https://fiddle.tiny.cloud/yiiaab/7
Without  `*[contentEditable=false] *[contentEditable=true] `rule  I can not select content of nested `cet` element

Comment (TheSpyder) [reply modules/oxide/src/less/theme/content/contenteditable/contenteditable.less:43]: I can select text in both strong tags, but I'm running Safari 16 beta. I guess it'll be another year or so before we can rely on that. Maybe leave a note about it in a comment? Doesn't seem worth making a task for.

**Review by lorenzo-pomili — APPROVED** [review]

**Pull request merged**